package.json

```diff
@@ -1,18 +1,15 @@
 {
 	"private": true,
 	"scripts": {
-		"lerna": "./node_modules/.bin/lerna",
-		"bootstrap": "npm run lerna -- bootstrap --hoist --nohoist=ts-node",
-		"gulp": "npm run lerna -- exec ./node_modules/.bin/gulp",
-		"tsc": "npm run gulp -- tsc",
-		"tslint": "npm run gulp -- tslint",
-		"test": "npm run gulp -- test",
-		"clean-build": "npm run gulp -- clean",
-		"clean-node_modules": "npm run lerna -- clean",
-		"clean-all": "npm run clean-build && npm run clean-node_modules"
+		"setup": "./node_modules/.bin/lerna exec npm install",
+		"bootstrap": "./node_modules/.bin/lerna bootstrap --force-local --hoist",
+		"compile": "./node_modules/.bin/lerna exec npm run compile",
+		"docs": "rm -rf ./api_docs && typedoc --out api_docs"
```

Comment:
typedoc should also use the local one: `./node_modules/typedoc` ?

Reply:
Ahhh, will update it thanks :)